server/py/services/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -837,7 +838,11 @@ def _delete_model_endpoint_monitoring_infra(
         for uid in uids:
             ModelMonitoringCurrentStatsFile(project=project, endpoint_id=uid).delete()
             ModelMonitoringDriftMeasuresFile(project=project, endpoint_id=uid).delete()
-            ModelMonitoringSchedulesFile(project=project, endpoint_id=uid).delete()
+            ModelMonitoringSchedulesFileEndpoint(
+                project=project, endpoint_id=uid
+            ).delete()
+
+        ModelMonitoringSchedulesFileChief(project=project).delete()
```

Comment:
Not sure this is the right place to delete it… We’re currently calling this function on each MEP deletion.
I think it would be better to handle the deletion inside the `delete_model_endpoint_monitoring_resources` method, since that only runs when the user deletes the project.
Alternatively, you could consider deleting it when calling `disable_model_monitoring`, but that approach is a bit riskier.